guides/software-versioning.md

```diff
@@ -43,6 +43,17 @@ version. This should be enforced by automated tools where practical (e.g.,
 the Python package templates require a "release" tag to be set, and scripts
 automatically use that to append "dev" to the version).
 
+### DOI For Software Releases
+
+Our open source software available to the public should include a DOI (Digitial
+Object Identifier) that has now become the standard for journal articles.
+Including a DOI for each released verison of software will enable (and
+encourage!) the public to reference our software and even more the specific
+version they used.  [Zenoodo](https://zenodo.org) enables the automatic DOI
+creation when a release is done on GitHub.  The place to start is by [logging
```

Comment:
is done on Github and this should be enabled by following this [GitHub guide](https://guides.github.com/activities/citable-code/).

Reply:
Actually that link is right at the bottom of the paragraph. :)